docs_app/content/deprecations/scheduler-argument.md

```diff
@@ -0,0 +1,101 @@
+
+<div class="alert is-important">
+    <span>
+        This deprecation was introduced in **RxJS 6.5** and will become breaking with **RxJS 8**.
+    </span>
+</div>
+
+## Operators affected by this Change
+
+- [from](/api/index/function/from)
+- [of](/api/index/function/of)
+- [merge](/api/index/function/merge)
+- [concat](/api/index/function/concat)
+- [startWith](/api/operators/startWith)
+- [endWith](/api/operators/endWith)
+- [combineLates](/api/index/function/combineLatest)
+
+## How to Refactor
+
+If you use any other operator from the list above and using the `scheduler` argument, you have to three potential refactoring options.
+
+### Refactoring of `of` and `from`
+
+`scheduled` is kinda copying the behavior of `from`. Therefore if you used `from`  with a `scheduler` argument, you can just replace them.
+
+For the `of` creation function you need to this Observable with `scheduled` and instead of passing the `scheduler` argument to `of` pass it to `scheduled`.
+Following code example demonstrate this process.
+
+```ts
+import { of, asyncScheduler, scheduled } from 'rxjs'; 
+
+// Deprecated approach
+of([1,2,3], asyncScheduler).subscribe(x => console.log(x));
+// suggested approach
+scheduled(of([1,2,3]), asyncScheduler).subscribe(x => console.log(x));
+```
+
+### Refactoring of `merge`, `concat`, `startWith` and `endWith`
+
+In case you used to pass a scheduler argument to one of these operators you probably had code like this:
+
+```ts
+import { concat, of, asyncScheduler } from 'rxjs'; 
+
+concat(
+  of('hello '),
+  of('World'),
+  asyncScheduler
+).subscribe(x => console.log(x));
+```
+
+To work around this deprecation you can leverage the [`scheduled`](/api/index/function/scheduled) function.
+
+```ts
+import { scheduled, of, asyncScheduler } from 'rxjs'; 
+import { concatAll } from 'rxjs/operators'
+
+scheduled(
+  [of('hello '), of('World')],
+  asyncScheduler
+).pipe(
+  concatAll()
+).subscribe(x => console.log(x));
+```
+
+You can apply this pattern to refactor deprecated usage of `concat`, `startWith` and `endWith` but do notice that you want to use [mergeAll](/api/operators/mergeAll) to refactor the deprecated usage of `merge`.
+
+### Refactoring `combineLatest`
+
+Unfortunately, there's no point in using `scheduled` with `combineLatest`. You rather want to leverage [`subscribeOn`](/api/operators/subscribeOn) or [`observeOn`](/api/operators/observeOn) depending on your goals. 
+
+E.g. code that used to look like this:
+
+```ts
+import { combineLatest, timer, asyncScheduler } from 'rxjs'; 
+
+combineLatest(
+  [timer(0, 1000), timer(500, 500)],
+  asyncScheduler
+).subscribe(console.log)
+```
+
+would become:
+
+```ts
+import { combineLatest, timer, asyncScheduler } from 'rxjs'; 
+import { subscribeOn } from 'rxjs/operators'
+
+combineLatest(
+  [timer(0, 1000), timer(500, 500)],
+).pipe(
+  subscribeOn(asyncScheduler)
+).subscribe(console.log)
+```
```

Comment:
Just realised that there's a `combineAll` operator that's the higher-order version of `combineLatest`, so the equivalent ought to be:

```ts
import { scheduled, of, asyncScheduler } from 'rxjs'; 
import { combineAll } from 'rxjs/operators'
scheduled(
  [timer(0, 1000), timer(500, 500)],
  asyncScheduler
).pipe(
  combineAll()
).subscribe(x => console.log(x));
```

And maybe this example should use the `[of('hello '), of('World')]` sources that were used above? Was there a particular reason for using the timers?